src/components/Accordion/Accordion.stories.tsx

```diff
@@ -21,7 +21,7 @@ export default {
 const AccordionListTemplate = ({ count, children, ...args }) => (
     <List type={ListTypes.Unordered} modifiers={["no-list-styling"]}>
         {range(count).map((i) => (
-            <li>
+            <li key={i}>
```

Comment:
I think it's not advised to use the iterator for keys (https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318).  Is it reasonable to ask accordion children to have ids?

Reply:
edit: oh my bad, I thought this was accordion.tsx rather than accordion.stories.tsx.  It's ok here since the data is static and in the story -- ignore me =)